src/clean_env_before_mining.py

```diff
@@ -10,6 +10,7 @@
 import os
 import shutil
 
+DATA_DIRECTORY = "../data"
 
 def clean_directory_content(script_dir: str, directory: str) -> None:
```

Comment:
```suggestion

def clean_directory_content(script_dir: str, directory: str) -> None:
```

Reply:
Here your are suggesting the same code, like is already in. Code refactoring for `clean_env_before_mining.py` will be part of other issue.